packages/ui/test/working-groups/components/MyEarningsStat.test.tsx

```diff
@@ -0,0 +1,60 @@
+import { screen, render } from '@testing-library/react'
+import BN from 'bn.js'
+import React from 'react'
+import { MemoryRouter } from 'react-router'
+
+import { MembershipContext } from '@/memberships/providers/membership/context'
+import { MyEarningsStat } from '@/working-groups/components/MyEarningsStat'
+import { UseMyEarnings } from '@/working-groups/hooks/useMyEarnings'
+
+let mockEarnings: UseMyEarnings
+
+jest.mock('../../../src/working-groups/hooks/useMyEarnings', () => {
+  return {
+    useMyEarnings: () => mockEarnings,
+  }
+})
```

Comment:
Mocking the hook gives us little value because it will test only the part of reading the hook and displaying returned values.

I would use a mock server here as in other tests.